### DISCUSSION TITLE
Q: Is there a list that maps the old IDs to the new ones?

### DISCUSSION THREAD

**Discussion by richstokes** [aquasecurity/tfsec#1481] [Help and suppports] [answered]
We have a bunch of excludes that use the legacy AWSxxx naming, how can I easily convert these to the new long-string-name format?

**Comment by owenrumney** [comment, marked as answer]
Hey @richstokes you can use tfsec --migrate-ignores <pathToTerraform> to update them all

**Comment by richstokes** [comment]
Oh, nice! Thank you!
